MethodRealization.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Goro.h"
+
```

Comment:
Please do not forget to rename all methods and variables which you renamed in header file(Goro.h)